pkg/providers/providersutils.go

```diff
@@ -271,14 +276,35 @@ func GetCurrentAzSubscriptionId() []string {
 		}
 	}
 
-	getAccountCmd := exec.Command("az", "account", "show", "--query", "[id]")
-	out, err := getAccountCmd.CombinedOutput()
+	getAccountCmdOne := exec.Command("az", "account", "list", "--all", "--query", "[].{id: id, name: name}")
+	out1, err := getAccountCmdOne.CombinedOutput()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	var so []SubOp
+	json.Unmarshal(out1, &so)
+
+	getAccountCmdTwo := exec.Command("az", "account", "show", "--query", "[id]")
+	out2, err := getAccountCmdTwo.CombinedOutput()
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	var ids []string
-	json.Unmarshal(out, &ids)
+	var id []string
+	json.Unmarshal(out2, &id)
+
+	subs := make([]string, len(so))
+
+	for i := range so {
+		subscription := fmt.Sprintf("%s (%s)", so[i].Name, so[i].ID)
+
+		if so[i].ID == id[0] {
```

Comment:
instead of indexing into `so` each time, you can access the value with `for _,v ...` and then use `v` her e